ambry-tools/src/main/java/com.github.ambry/store/DumpLogTool.java

```diff
@@ -80,10 +80,10 @@ public DumpLogTool(VerifiableProperties verifiableProperties, StoreToolsMetrics
     this.clusterMap =
         ((ClusterAgentsFactory) Utils.getObj(clusterMapConfig.clusterMapClusterAgentsFactory, clusterMapConfig,
             hardwareLayoutFilePath, partitionLayoutFilePath)).getClusterMap();
-    if (blobsPerSec > 0) {
-      this.throttler = new Throttler(blobsPerSec, 1000, true, SystemTime.getInstance());
-    } else if (blobsPerSec < 0) {
-      throw new IllegalArgumentException("BlobsPerSec " + blobsPerSec + " cannot be negative ");
+    if (bytesPerSec > 0) {
+      this.throttler = new Throttler(bytesPerSec, 1000, true, SystemTime.getInstance());
+    } else if (bytesPerSec < 0) {
```

Comment:
you can ensure this when you load the property by providing a range from 0 to Integer MAX